kernel/rros/clock.rs

```diff
@@ -579,7 +536,7 @@ fn timer_needs_enqueuing(timer: *mut RrosTimer) -> bool {
     }
 }
 
-//rq相关未测试，其余测试通过
+// This function have passed other tests without rq related tests.
```

Comment:
Grammatical errors and unclear expression.

Reply:
Thanks.